configs/datasets.yaml

```diff
@@ -23,3 +23,232 @@ experiment_groups:
       B:
         drop: 0.75
         transforms: null
+  none-vs-transforms:
```

Comment:
can/should we separate out into multiple groups?

Reply:
Have split them out into several smaller groups